src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/SpatialPoint.java

```diff
@@ -36,4 +37,24 @@ public class SpatialPoint extends Spatial {
         builder.addSubfield(SPATIAL_POINT_X, point.getX());
         builder.addSubfield(SPATIAL_POINT_Y, point.getY());
     };
+
+    public static boolean isPoint(Node node) {
+        var hasChild = hasChildNode(node, "gml:Point/gml:pos");
+        var srsName = Base.getAttribute(node, "srsName")
+            .map(Node::getTextContent)
+            .filter(StringUtils::isNotBlank)
+            .orElse(null);
+
+        if (!hasChild) {
+            return false;
+        }
+
+        // TS002: if @srsName=http://www.opengis.net/def/crs/EPSG/0/28992
+        if (RD_SRS_NAME.equals(srsName)) {
+            return true;
+        }
+
+        // TS003: if @srsName not present OR @srsName=http://www.opengis.net/def/crs/EPSG/0/4326
+        return srsName == null || LONLAT_SRS_NAME.equals(srsName);
```

Comment:
See new clarifying comment in the issue: also check that X and Y are both present. Same for the corners of a box.

Reply:
So this is already done in the getPoint method in the Spatial class. The current behavior is that if a point (or a box) is validated according to the rules in the `isPoint` method, it will try to parse these 2 values (for boxes it does it twice, for 2 corners). 

If the values are not valid numbers, it will throw a runtime exception. The bag validator service should already have validated these values and yield errors if they are not correct.

If we filter out invalid values from the list (because thats what the isPoint method is used for) we are silently discarding invalid values.

Reply:
My 'keep the barbarians at the gate' hat might be too strong.

I see the `BagValidatorRule pointsHaveAtLeastTwoValues` method is concerned about normal points and corners. But then, shouldn't the `srsName`s  also be validated? That would overrule this issue. Not sure yet.